src/api/json/catalog.json

```diff
@@ -737,6 +737,18 @@
       "fileMatch": ["commands.json"],
       "url": "https://json.schemastore.org/commands.json"
     },
+    {
+      "name": "Common Catalog Schema",
+      "description": "Universal schema for all catalog data focused on transformations and relationships",
+      "fileMatch": [
+        "*.cat.json",
+        "*.catalog.json",
+        "catalog.json",
```

Comment:
catalog.json/yaml are very generic file names. I suggest you either remove these or change them to something less generic

Reply:
Thanks! Done!